recipes/interact.js

```diff
@@ -0,0 +1,98 @@
+let ingSet = false;
+let insSet = false;
+let recipeInfo = {};
+
+function recipeToJSON() {
+    recipeInfo.recipeName = document.getElementById('recipeName').innerHTML;
+    recipeInfo.recipeDescription = document.getElementById('recipeDescription').innerHTML;
```

Comment:
if you don't want to include things like <br /> or /n in your description, try using .textContents as opposed to .innerHTML